frontend/mr-admin-flate/src/components/avtaler/AvtalePersonvernForm.tsx

```diff
@@ -0,0 +1,135 @@
+      </div>
+    );
+  }
+
+  return (
+    <VStack gap="4" className={styles.container}>
+      <GuidePanel poster className={styles.guide_panel}>
+        Huk av de personopplysningene som er avtalt i databehandleravtalen. NAV tiltaksenhet/fylke
+        er ansvarlig for at listen er i samsvar med gjeldende databehandleravtale.
+      </GuidePanel>
+      <HGrid columns={2}>
+        <VStack>
+          <Label size="small">Opplysninger om brukeren som alltid kan/må behandles</Label>
+          <BodyShort size="small" textColor="subtle">
+            Fjern avhukingen hvis noen av opplysningene ikke er relevante for denne avtalen.
+          </BodyShort>
+          {tiltakstype?.personopplysninger?.ALLTID.map((p) => (
+            <Checkbox
+              checked={watchPersonopplysninger.includes(p)}
+              onChange={() =>
+                setValue("personopplysninger", addOrRemove(watchPersonopplysninger, p))
+              }
+              size="small"
+              key={p}
+            >
+              {personopplysningToTekst(p)}
+            </Checkbox>
+          ))}
+        </VStack>
+        <VStack justify="space-between">
+          <VStack>
+            <Label size="small">
+              Opplysninger om brukeren som ofte er nødvendig og relevant å behandle
+            </Label>
+            <BodyShort size="small" textColor="subtle">
+              Huk av for de opplysningene som er avtalt i databehandleravtalen.
+            </BodyShort>
+            {tiltakstype?.personopplysninger?.OFTE.map((p) => (
+              <Checkbox
+                checked={watchPersonopplysninger.includes(p)}
+                onChange={() =>
+                  setValue("personopplysninger", addOrRemove(watchPersonopplysninger, p))
+                }
+                size="small"
+                key={p}
+              >
+                {personopplysningToTekst(p)}
+              </Checkbox>
+            ))}
+          </VStack>
+          <VStack>
+            <Label size="small">
+              Opplysninger om brukeren som sjelden eller i helt spesielle tilfeller er nødvendig og
+              relevant å behandle
+            </Label>
+            <BodyShort size="small" textColor="subtle">
+              Huk av for de opplysningene som er avtalt i databehandleravtalen.
+            </BodyShort>
+            {tiltakstype?.personopplysninger?.SJELDEN.map((p) => (
+              <Checkbox
+                checked={watchPersonopplysninger.includes(p)}
+                onChange={() =>
+                  setValue("personopplysninger", addOrRemove(watchPersonopplysninger, p))
+                }
+                size="small"
+                key={p}
+              >
+                {personopplysningToTekst(p)}
+              </Checkbox>
+            ))}
+          </VStack>
+          <BodyShort size="small">
+            *Se egne retningslinjer om dette i veileder for arbeidsrettet brukeroppfølging pkt. 4.4.
```

Comment:
Skulle vi sjekket med Tonje eller Grethe Hagland om det finnes et sted på NAVet der veileder for arbeidsrettet brukeroppfølging pkt 4.4 ligger så kunne vi lenket til det fra her?